fvm/evm/stdlib/contract.cdc

```diff
@@ -21,27 +21,27 @@ contract EVM {
         // height or number of the block
         height: UInt64,
         // hash of the block
-        hash: String,
+        hash: [UInt8],
         // timestamp of the block creation
         timestamp: UInt64,
         // total Flow supply
         totalSupply: Int,
         // all gas used in the block by transactions included
         totalGasUsed: UInt64,
         // parent block hash
-        parentHash: String,
+        parentHash: [UInt8],
```

Comment:
shouldn't here be `Uint8;32` as well?

Reply:
Yeah, this was suggested by Ramtin too and was changed in the dependent PR, #6221